resources/views/admin/contests/show.blade.php

```diff
@@ -72,11 +72,13 @@
             <dd class="contest">
                 <div class="contest__description">{!! markdown($contest->description_enter) !!}</div>
             </dd>
+            <br />
             <dt class="admin-contest__meta-row">Voting Description</dt>
             <dd class="contest">
                 <div class="contest__description">{!! markdown($contest->description_voting) !!}</div>
             </dd>
             @if ($contest->extra_options !== null)
+                <br />
```

Comment:
`<dl>` may not contain `<br />`...

[Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)